gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -330,13 +335,15 @@ nnsconf_loadconf (gboolean force_reload)
     /* File not found or not configured */
     g_free (conf.conffile);
 
-    /* Try to read from Environmental Variables */
-    conf.conffile = _strdup_getenv (NNSTREAMER_ENVVAR_CONF_FILE);
+    if (g_path_is_absolute (NNSTREAMER_DEFAULT_CONF_FILE) &&
```

Comment:
g_path_is_absolute (NNSTREAMER_DEFAULT_CONF_FILE) is always true

Reply:
Got it